cove_bods/templates/cove_bods/additional_checks_table.html

```diff
@@ -658,10 +657,11 @@
             {# Currently this applies to 0.4+ #}
             <tr>
                 <td>
-                    {% blocktrans %}This Statement relates to the subject of a declaration but a subject must be an entity or person.{%endblocktrans%}
+                    {% blocktrans %}<code>declarationSubject</code> must be an entity or person. Check that <code>recordId</code> values are being correctly generated and used.{% endblocktrans %}
```

Comment:
```suggestion
                    {% blocktrans %}<code>declarationSubject</code> must reference an entity or person. Check that <code>recordId</code> values are being correctly generated and used.{% endblocktrans %}
```
assume we've avoiding saying "must be an entity or person statement" because it might be a record id or a statement id?